.azure-devops/templates/set-testenv-sentinel.yml

```diff
@@ -41,6 +43,8 @@ steps:
       AZEXT_IOT_TESTDPS: $(azext_iot_testdps)
       AZEXT_IOT_TESTSTORAGEURI: $(azext_iot_teststorageuri)
```

Comment:
You said

> the SAS URL can be generated using the connection string and container name

So why do we still have code that relies on `azext_iot_teststorageuri`?